test/functional/fixtures/api/es-next/assertions/test.js

```diff
@@ -4,7 +4,8 @@ describe('[API] Assertions', function () {
     it('Should perform .eql() assertion', function () {
         return runTests('./testcafe-fixtures/assertions-test.js', '.eql() assertion', {
             shouldFail: true,
-            only:       'chrome'
+            only:       'chrome',
```

Comment:
I guess we should not change the current behavior.
For you case, check the expected value using [config](https://github.com/DevExpress/testcafe/pull/4349/files#diff-47530cfffc057e1dfc7fb899799c1694R3).